gnucash/import-export/test/gmock-qofquery.cpp

```diff
@@ -0,0 +1,68 @@
+#include <config.h>
```

Comment:
Using the terms defined in [GMock docs](https://github.com/google/googletest/blob/master/googlemock/docs/for_dummies.md#what-is-gmock) this is a *fake*-qofquery rather than a *mock*-qofquery, and `gmock` doesn't have anything to do with it.

Reply:
Ok, I'll check that

Reply:
`fake-qofquery.h`?

Reply:
Done in PR #738